sdk/program/src/message/legacy.rs

```diff
@@ -58,6 +58,11 @@ lazy_static! {
     };
 }
 
+#[deprecated(
+    since = "2.0.0",
+    note = "please use solana_sdk::reserved_account_keys::ReservedAccountKeys instead"
```

Comment:
nit: how about saying exactly what to use?
```suggestion
    note = "please use solana_sdk::reserved_account_keys::ReservedAccountKeys::is_reserved instead"
```